srcpkgs/embree/template

```diff
@@ -4,6 +4,7 @@ version=2.17.7
 revision=2
 archs="x86_64"
 build_style=cmake
+configure_args="-DTBB_ROOT=/usr -DEMBREE_IGNORE_CMAKE_CXX_FLAGS=OFF"
```

Comment:
@thypon

Reply:
I think it derived from archlinux package.

It should be defined this way now '-DEMBREE_MAX_ISA="AVX512SKX"'